comp/otelcol/otlp/components/processor/tagenrichmentprocessor/testdata/config_logs_strict.yaml

```
@@ -0,0 +1,9 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2024-present Datadog, Inc.
+
+tagenrichment/empty:
+  logs:
+    log:
+     aa
```

This works because of a quirk on our configuration loading system which I am looking to fix in the short term. The type is `[]string` so we should put an array here
```suggestion
    log:
      - aa
```